### PR TITLE
Import BFO

### PR DESCRIPTION
**Pull request by gouttegd**
This PR replaces the manually maintained BFO components (`bfo-classes-mininal.owl` and `bfo-axioms.owl`) by a properly declared, ODK-style BFO import module.

This ensures that the BFO-owned classes distributed with RO are always kept in sync with their original BFO versions, as discussed in #107.

### CODE REVIEW

**Comment by wdduncan** [comment]
Thanks @gouttegd !
I hold off any more work on [owns / is owned by](https://github.com/oborel/obo-relations/pull/705) until this is merged.

**Comment by wdduncan** [comment]
@gouttegd I'm adding:
Fixes #107 

Delete this comment if you think this PR doesn't close the issue.

File: src/ontology/ro-odk.yaml
Comment (matentzn) [inline src/ontology/ro-odk.yaml:29]:
```suggestion
      module_type: custom
```

File: src/ontology/ro.Makefile
Comment (matentzn) [inline src/ontology/ro.Makefile:77]:
```suggestion

# We are not using the BFO PURL here, just two manually curated BFO components.
imports/bfo_import.owl: components/bfo-axioms.owl components/bfo-axioms.owl
	$(ROBOT) merge -i components/bfo-axioms.owl -i components/bfo-axioms.owl -o $@
```

File: src/ontology/ro.Makefile
Comment (matentzn) [inline src/ontology/ro.Makefile:32]: Lets not make any updates to core release better..

**Review by matentzn — CHANGES_REQUESTED** [review]

File: src/ontology/ro.Makefile
Comment (gouttegd) [reply src/ontology/ro.Makefile:77]: Err, sorry, but what good does that make? This does not fix anything, _unless_ we also update once the `components/bfo-axioms.owl` to synchronize it with the current version of upstream BFO – in which case this PR becomes an implementation of “solution 2” [described here](https://github.com/oborel/obo-relations/issues/107#issuecomment-1485063040). This just turns the manually curated components into a manually curated pseudo-import.

Comment (gouttegd) [reply src/ontology/ro.Makefile:77]: I’d argue it’s even worse than the current situation, but this would make it _seem_ like we are importing upstream BFO, while in reality we are still using a manually curated frozen (and outdated!) version of it.

Comment (matentzn) [reply src/ontology/ro.Makefile:77]: Sorry I misunderstood something in slack - now I got it. If I understand it correctly and purl.../bfo.owl in its current form actually more or less corresponds to a merge of bfo-axioms with bfo-classes-minimal, we should go ahead with what you had. Sorry about the confusion!

Comment (gouttegd) [reply src/ontology/ro.Makefile:77]: > purl.../bfo.owl in its current form actually more or less corresponds to a merge of bfo-axioms with bfo-classes-minimal

Yes. Except that the version published by the PURL server has updated definitions, while our manually maintained copy (or not so maintained actually) still has definitions from several years ago – hence the incompatibility!

The whole point of this PR is to use whatever version of BFO is currently published by the PURL server, so that when third-party ontologies want to import both purl.obolibrary.org/obo/ro.owl and purl.obolibrary.org/obo/bfo.owl, they can do so without being bitten by the fact that RO comes with its own outdated definitions of BFO classes.

**Comment by gouttegd** [comment]
Closing here as it was decided to fix using the simpler route in #709.

**Pull request closed without merging**
